arangod/GeneralServer/ListenTask.cpp

```diff
@@ -122,5 +119,5 @@ void ListenTask::stop() {
 
   _bound = false;
   _acceptor->close();
-  _acceptor.reset();
+//  _acceptor.reset();
```

Comment:
Is this intentional?

Reply:
Good question... I commented this out because it was crashing with the reset, at least at some point during my changes. Will now check and confirm whether the `reset` indeed needs to be removed or if it can go back in.